spotify_os.xcconfig

```diff
@@ -10,6 +10,8 @@ ENABLE_NS_ASSERTIONS_Debug = YES
 ENABLE_NS_ASSERTIONS_Release = NO
 ONLY_ACTIVE_ARCH_Debug = YES
 ONLY_ACTIVE_ARCH_Release = NO
+ONLY_ACTIVE_ARCH = $(ONLY_ACTIVE_ARCH_$(CONFIGURATION))
+SUPPORTED_PLATFORMS = macosx iphoneos iphonesimulator watchsimulator tvsimulator
```

Comment:
I think it might be good to add `CLANG_MODULES_AUTOLINK = NO` as well. So that we don’t add dependencies on other modules accidentally. I.e. making sure it requires thought.